Library/Formula/bazel.rb

```diff
@@ -0,0 +1,63 @@
+class Bazel < Formula
+  desc "Google's own build tool"
+  homepage "http://bazel.io/"
+  url "https://github.com/bazelbuild/bazel/archive/0.1.1.tar.gz"
+  sha256 "49d11d467cf9e32dea618727198592577fbe76ff2e59217c53e3515ddf61cd95"
+
+  depends_on :java => "1.8+"
+
+  # Replace the default system wide rc path to /usr/local/etc/bazel/bazel.bazelrc.
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!